jobs/pipelines/powervs/powervs-image-create-and-upload-job/Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
 	    string(defaultValue: '', description: 'Rhel Url', name: 'RhelUrl')
 	    string(defaultValue: '', description: 'Rhcos Url', name: 'RhcosUrl')
 	    string(defaultValue: '', description: 'Centos Url', name: 'CentosUrl')
-        string(defaultValue: 'v0.1.1-alpha.6', description: "Pvsadm version", name: 'pvsadmVersion')
+        string(defaultValue: 'v0.1.2-alpha.2', description: "Pvsadm version", name: 'pvsadmVersion')
```

Comment:
can you please fix this indentation ?

Reply:
@sudeeshjohn Fixed it https://github.com/ppc64le-cloud/jenkins-infra/blob/95f88d45de50a02a45e19daf4857f043430d24dc/jobs/pipelines/powervs/powervs-image-create-and-upload-job/Jenkinsfile